pmm-app/src/pmm-settings/components/SignUp/SignUp.validators.ts

```diff
@@ -0,0 +1,20 @@
+import { Messages } from './SignUp.messages';
+
+export const required = (value: string | boolean | undefined) => (
+  value ? undefined : Messages.errors.requiredField
+);
+
+export const validEmail = (value: string) => {
+  const emailRe = RegExp(''
```

Comment:
Let's try to get away with regex literal using forward slashes only. The constructor wouldn't be that justified here )

Reply:
eslint should be have, or let teach it to be )

Reply:
I agree, this was an attempt to avoid having a really long line, since it's not practical to break a regex literal on several lines. Do you have an advice for that?

Reply:
yes, it should just be a regex, although a bit long )

Reply:
Done, eslint's `max-len` will ignore regex literals